proto/frequenz/api/microgrid/microgrid.proto

```diff
@@ -490,3 +512,61 @@ message Connection {
 message ConnectionList {
   repeated Connection connections = 1;
 };
+
+// Parameters for setting the active power of an appropriate component using the
+// `SetPowerActive` RPC.
+message SetPowerActiveParam {
+  // The ID of the component to set the output active power of.
+  uint64 component_id = 1;
+
+  // The output active power level, in watts.
+  // -ve values are for discharging, and +ve values are for charging.
+  float power = 2;
+}
+
+// Parameters for setting the reactive power of an appropriate component using
+// the `SetPowerReactive` RPC.
+message SetPowerReactiveParam {
+  // The ID of the component to set the output reactive power of.
+  uint64 component_id = 1;
+
+  // The output reactive power level, in VAr.
+  // -ve values are for inductive (lagging) power , and +ve values are for
+  //  capacitive (leading) power.
+  float power = 2;
+}
+
+// Parameters for setting bounds of a given metric of a given component.
+message SetBoundsParam {
+  // An enumerated list of metrics whose bounds can be set.
+  enum TargetMetric {
+    TARGET_METRIC_UNSPECIFIED = 0;
+    TARGET_METRIC_POWER_ACTIVE = 1;
+    TARGET_METRIC_CURRENT = 2;
+    TARGET_METRIC_CURRENT_PHASE_1 = 3;
+    TARGET_METRIC_CURRENT_PHASE_2 = 4;
+    TARGET_METRIC_CURRENT_PHASE_3 = 5;
+    TARGET_METRIC_POWER_REACTIVE = 6;
+  }
+
+  // The ID of the target component.
+  uint64 component_id = 1;
+
+  // The target metric whose bounds have to be set.
+  TargetMetric target_metric = 2;
+
+  // The bounds for the target metric.
+  frequenz.api.common.v1.metrics.Bounds bounds = 3;
+}
+
+// Encapsulation of a component ID, intended to be used as a parameter for rpc
```

Comment:
There are plenty of new `component_id` fields introduced here with the type `uint64`. Should these be `ComponentIDParam`? If not, I think the docs here (and for sensor below) should be clarified that these are only intended to be used as top-level parameters for the rpc methods themselves, and perhaps a little blurb explaining why.

Reply:
This will go away once https://github.com/frequenz-floss/frequenz-api-microgrid/issues/149 is done. Is it okay if I handle this directly there?